### PR TITLE
Improve read and add write

### PR DESCRIPTION
**Pull request by sjorge**
I mostly have stuff similar to this added when working on a new device.
Might be useful to have them around in general, should also make it easier to ask people to try stuff when having problems to be able to write a raw payload to a cluster.

### CODE REVIEW

File: converters/toZigbee.js [hunk 1]
```diff
@@ -26,8 +26,20 @@ const converters = {
...
+                const state = {};
+                state[value.store] = result;
+                return {state: state};
+            }
+        },
+    },
+    write: {
+        key: ['write'],
+        convertSet: async (entity, key, value, meta) => {
+            await entity.write(value.cluster, value.payload, utils.getOptions(meta.mapped, entity));
```
Comment (Koenkk) [inline converters/toZigbee.js:41]: Please also allow to pass options here similar to the `read`.

Comment (sjorge) [reply converters/toZigbee.js:41]: Do we merge them with the getOptions or do we pass them in favor of getOptions?

File: converters/toZigbee.js [hunk 2]
```diff
@@ -26,8 +26,20 @@ const converters = {
     read: {
         key: ['read'],
         convertSet: async (entity, key, value, meta) => {
-            const result = await entity.read(value.cluster, value.attributes);
+            const result = await entity.read(value.cluster, value.attributes, (value.hasOwnProperty('options') ? value.options : {}));
             meta.logger.info(`Read result of '${value.cluster}': ${JSON.stringify(result)}`);
+            if (value.hasOwnProperty('store')) {
+                const state = {};
+                state[value.store] = result;
+                return {state: state};
```
Comment (Koenkk) [inline converters/toZigbee.js:34]: Can be simplified to

```js
return {state: {[value.store]: result}};
```

If I understand correctly, the goal is to expose the result on a specific property? Not sure if `store` is a good name.

Comment (sjorge) [reply converters/toZigbee.js:34]: Yes, open to suggestions on the name. I’m not happy with store either but it was the best I could come up with to ‘store’ the result in the state.

Comment (sjorge) [reply converters/toZigbee.js:34]: I called it 'state_property' instead, not sure it's a huge improvement. I did simplyfy the code.

File: converters/toZigbee.js (hunk 1 above)
Comment (sjorge) [reply converters/toZigbee.js:41]: Something like the new commit?

Comment (Koenkk) [reply converters/toZigbee.js]: Looks good, thanks!

**Pull request merged**